ramls/order.raml

```diff
@@ -228,17 +228,23 @@ resourceTypes:
                 example:
                   strict: false
                   value: !include examples/errors_400.sample
+              text/plain:
+                example: "unable to create Piece -- Bad request"                  
           404:
               description: "Request id does not exists"
               body:
                 application/json:
+                text/plain:
```

Comment:
Not sure if `404` can be with `text/plain` (the same for delete)

Reply:
Ok. It makes sense because the request will fail before that.